tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rules/FoldSetRule.scala

```diff
@@ -113,31 +114,4 @@ class FoldSetRule(rewriter: SymbStateRewriter, renaming: IncrementalRenaming) ex
     case _ =>
       throw new RewriterException("%s is not applicable".format(getClass.getSimpleName), state.ex)
   }
-
-  // To guarantee evaluation uniqueness, we will need to compare cells.
-  // To that end, we pre-cache equality constraints
-  def cacheEqualityConstraints(initialState: SymbState, cells: Seq[ArenaCell]): SymbState = {
```

Comment:
Why is this code removed? Was it useless?

Reply:
It was never called.